### ISSUE TITLE
LastPasswordSet date has not the right time in certain case

### ISSUE THREAD

**Issue by skidrow88** [adrecon/ADRecon#20]
I noticed on certain operating systems, when the output format date is '%m/%d/%Y %H:%M:%S %p', there is a time difference about 2 hours between the lastPasswordSet and the whenCreated field.

However, this information is wrong because the user has never changed his password. Both values should be the same.  

<img width="304" alt="Capture d’écran 2022-11-15 à 11 47 04" src="https://user-images.githubusercontent.com/30598851/201900861-1dcfb804-af94-4cd8-a955-641922ca941a.png">

We check on our Active Directory and the LastPasswordSet value is 6/14/2016 1:25:44 PM. 

It could be problematic when we want to detect users have not changed their initial password by comparing lastPasswordSet and whenCreated.

Thanks in advance

**Comment by prashant3535** [comment]
Are you in GMT+2 Timezone by any chance ?

**Comment by skidrow88** [comment]
Yes, the Active Directory is located in Budapest, now GMT +1:00 but in June it's GMT +02:00

**Comment by prashant3535** [comment]
That could be the issue ... the tool doesn't do any Timezone modifications by itself ... still I'll go through the code to see if there is any solution.

**Comment by skidrow88** [comment]
It seems you use the Convert.ToDateTime for PasswordLastSet but not for whenCreated attribute.

`PasswordLastSet = Convert.ToDateTime(AdComputer.Members["PasswordLastSet"].Value);`

Thank you

**Comment by skidrow88** [comment]
Hello ! Any news on a possible fix ?
